packages/jest-cli/src/search_source.js

```diff
@@ -162,6 +162,18 @@ class SearchSource {
     };
   }
 
+  findTests(paths: Array<Path>): SearchResult {
+    require('fs').writeFileSync('/tmp/lep', paths);
+    return {
+      tests: toTests(
+        this._context,
+        paths
+          .map(p => path.resolve(process.cwd(), p))
+          .filter(this.isTestFilePath.bind(this)),
```

Comment:
What happens if it can't find any tests, what does the message look like and does it make sense?